miden-lib/src/transaction/inputs.rs

```diff
@@ -246,7 +248,15 @@ fn add_account_to_advice_inputs(
     let code = account.code();
 
     // extend the merkle store with account code tree
-    inputs.extend_merkle_store(code.procedure_tree().inner_nodes());
+    // inputs.extend_merkle_store(code.procedure_tree().inner_nodes());
+
+    // extend the advice_map with the account code data
+    // inputs.extend_map([(*code.procedure_commitment(), code.as_elements())]);
+
+    for (proc, _) in code.procedures() {
```

Comment:
Here I extend the advice map with a proc_root -> proc_elements, enabling us to query them from the internals of the VM.

Reply:
I'm not quite sure I follow what you are doin in this loop, but the code on line 254 looks correct. What is the reason for commenting it out?